### PR TITLE
feat: implement to_address

### LINKED ISSUE #289: feat: implement to_address
https://github.com/ethereum/execution-specs/blob/1adcc1bfe774798bcacc685aebc17bd9935078c3/src/ethereum/cancun/utils/address.py#L27

### PR DESCRIPTION
**Pull request by obatirou**
Close #289

### CODE REVIEW

File: cairo/ethereum_types/numeric.cairo [hunk 1]
```diff
@@ -20,6 +20,15 @@ struct U256 {
     value: U256Struct*,
 }
 
+struct UintU256Enum {
+    uint: Uint*,
+    u256: U256,
+}
+
+struct UintU256 {
```
Comment (ClementWalter) [inline cairo/ethereum_types/numeric.cairo:28]:
```suggestion
struct UnionUintU256 {
```

Comment (obatirou) [reply cairo/ethereum_types/numeric.cairo]: [19b348a](https://github.com/kkrt-labs/keth/pull/292/commits/19b348a1b36e9f855bf47b2381f4596f39584d0d)

File: cairo/ethereum/cancun/utils/address.cairo [hunk 2]
```diff
@@ -4,11 +4,42 @@ from starkware.cairo.common.cairo_builtins import BitwiseBuiltin, KeccakBuiltin
...
+from ethereum_types.numeric import Uint, UnionUintU256
 from ethereum.cancun.fork_types import Address
 from ethereum.crypto.hash import keccak256
 from ethereum.utils.numeric import divmod
-from src.utils.bytes import felt_to_bytes20_little, felt_to_bytes, uint256_to_bytes32_little
+
+from src.utils.bytes import (
+    felt_to_bytes20_little,
+    bytes_to_felt,
+    felt_to_bytes,
```
Comment (ClementWalter) [inline cairo/ethereum/cancun/utils/address.cairo:15]:
```suggestion
```

Comment (obatirou) [reply cairo/ethereum/cancun/utils/address.cairo:15]: used in compute_address below

**Review by enitrat — DISMISSED** [review]

**Review by ClementWalter — APPROVED** [review]

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**